src/components/DateTimePicker/DateTimePicker.test.jsx

```diff
@@ -3,7 +3,7 @@ import React from 'react';
 import { mount } from 'enzyme';
 import moment from 'moment';
 
-import __unstableDateTimePicker, {
+import DateTimePicker, {
```

Comment:
The /* eslint-disable react/jsx-pascal-case */ should be deleted in the first line of this file